bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/gems/CandidateAsterismsTreeTableModel.java

```diff
@@ -95,9 +96,11 @@ public Object getValue(Row row) {
         // (XXX There might be another way to get the parent node)
         private Row _parent;
 
+        private final Option<Long> _when;
+
         // A row corresponding to a GemsGuideStars object
         // (top level node, displays the Strehl info and checkbox)
-        Row(GemsGuideStars gemsGuideStars, List<Row> children) {
+        Row(GemsGuideStars gemsGuideStars, List<Row> children, Option<Long> when) {
```

Comment:
I assume AGS needs to get the coordinates of the targets at the given time, both the one in the observation and the ones from the catalog
Which takes me to another point, we should likely correct on the catalog side too, right?